### PR TITLE
Cleanup around date related columns + added OptionalLocalDateColumn

### PR DESCRIPTION
**Pull request by mmatloka**

### CODE REVIEW

File: phantom-dsl/src/test/scala/com/websudos/phantom/tables/OptionalPrimitivesCassandra22.scala [hunk 1]
```diff
@@ -37,7 +37,8 @@ import com.websudos.util.testing._
 case class OptionalPrimitiveCassandra22(
                                          pkey: String,
                                          short: Option[Short],
-                                         byte: Option[Byte]
+                                         byte: Option[Byte],
```
Comment (alexflav23) [inline phantom-dsl/src/test/scala/com/websudos/phantom/tables/OptionalPrimitivesCassandra22.scala]: Looks like indentation needs fixing here.

**Comment by mmatloka** [comment]
wait, I see some issues here ;)

File: phantom-dsl/src/test/scala/com/websudos/phantom/tables/OptionalPrimitivesCassandra22.scala (hunk 1 above)
Comment (mmatloka) [inline phantom-dsl/src/test/scala/com/websudos/phantom/tables/OptionalPrimitivesCassandra22.scala]: hm, intelij formatter does such indent, is there rule for this in scalastyle?

**Comment by mmatloka** [comment]
I will squash this branch once review will be finished ;)

File: phantom-dsl/src/test/scala/com/websudos/phantom/tables/OptionalPrimitivesCassandra22.scala (hunk 1 above)
Comment (alexflav23) [inline phantom-dsl/src/test/scala/com/websudos/phantom/tables/OptionalPrimitivesCassandra22.scala]: @mmatloka Have a look at [this](http://stackoverflow.com/questions/26880677/intellij-scala-plugins-case-class-indentation-is-absurd) for details.

**Comment by alexflav23** [comment]
Hi @mmatloka ,

I think we left this behind a bit, it looks like it's still useful. Can you pull the latest develop in so I can merge?

Regards.

**Comment by mmatloka** [comment]
I have quashed and rebased it on latest develop yesterday, but forgot to comment, but ok, there it is based on latest develop :).

**Pull request merged**